scripts/refreshKibanaIndex.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/sh
+
+ES_HEAD="localhost:9200"
+KIBANA_PREFIX="$ES_HEAD/.kibana/"
+
+CURL="curl -g"
+REFRESH="/_refresh"
+
+KIBANA_LOG_FILE="/var/log/probe/KibanaStartup.log"
+
+if [ ! -e $KIBANA_LOG_FILE ]; then
+   touch $KIBANA_LOG_FILE
+fi
+
+echo `date +'%D %T'` "   Refreshing Kibana index..." >> $KIBANA_LOG_FILE
+$CURL -XPOST "$KIBANA_PREFIX""$REFRESH""$PRETTY"     >> $KIBANA_LOG_FILE
```

Comment:
$PRETTY is not defined

Also: Please see my comment above. I see no need to complicate this by splitting the curl statement up in parts.